model/textparse/openmetricsparse_test.go

```diff
@@ -206,7 +234,7 @@ foo_total 17.0 1520879607.789 # {xx="yy"} 5`
 			v:    17,
 			lset: labels.FromStrings("__name__", "foo_total"),
 			t:    int64p(1520879607789),
-			e:    &exemplar.Exemplar{Labels: labels.FromStrings("xx", "yy"), Value: 5},
+			e:    &exemplar.Exemplar{Labels: labels.FromStrings("id", "counter-test"), Value: 5},
```

Comment:
Is this change covered by the commit description?  I couldn't see the relevance.

Reply:
The advantage of doing this is `yy` is not really descriptive but `counter-test` will give you a better idea about what you are testing and it is harder to make mistakes.

Reply:
Please, include this in the commit description, so future readers understand your intent.

Reply:
👍🏼 Done.